netmaster/master/endpointGroup.go

```diff
@@ -67,10 +73,14 @@ func CreateEndpointGroup(tenantName, networkName, groupName string) error {
 	// params for docker network
 	if GetClusterMode() == "docker" {
```

Comment:
https://github.com/contiv/netplugin/blob/c4075e98d9c7f841bb89a5202ab5e3fc7c8c3287/netmaster/master/netmaster.go#L48

We have only these values for cluster mode. This is cosmetic issue. I will take care of in next set of PRs.